File: code/frontend/ios-app/OptiFit/OptiFit/Models/Profile/UserStatsDto.swift
struct UserStatsDto: Codable {
    let activeDays: Int
    let totalWorkouts: Int
    let totalExercises: Int
    let totalSets: Int
    let totalReps: Int
    let totalWeight: Int
    let totalDuration: Int
    let totalCalories: Int
    let workoutStreak: Int
    let averageDuration: Int
}

File: code/backend/src/application/qb8s.net.OptiFit.CQRS/Dtos/User/UserStatsDto.cs
namespace qb8s.net.OptiFit.CQRS.Dtos.User;

public class UserStatsDto
{
    public int ActiveDays { get; set; }
    public int TotalWorkouts { get; set; }
    public int TotalExercises { get; set; }
    public int TotalSets { get; set; }
    public int TotalReps { get; set; }
    public int TotalWeight { get; set; }
    public int TotalDuration { get; set; }
    public int TotalCalories { get; set; }
    public int WorkoutStreak { get; set; }
    public int AverageDuration { get; set; }
}